app/routes/circulars._archive._index/DateSelectorMenu.tsx

```diff
@@ -77,7 +78,11 @@ export function DateSelector({
   defaultStartDate?: string
   defaultEndDate?: string
 }) {
-  const [showContent, setShowContent] = useState(false)
+  const dateSelectorRef = useRef<HTMLDivElement>(null)
+  const [showDateSelector, setShowDateSelector] = useState(false)
```

Comment:
Renaming of the state from `showContent` to `showDateSelector` makes this a larger diff than necessary. Please keep the old name.